package-lock.json

```diff
@@ -37822,15 +37822,15 @@
       "version": "0.54.0",
       "license": "Apache-2.0",
       "dependencies": {
-        "@opentelemetry/instrumentation": "^0.208.0"
+        "@opentelemetry/instrumentation": "^0.208.0",
+        "@opentelemetry/semantic-conventions": "^1.37.0"
```

Comment:
For this monumentus change, thou shalt pay with 4h of CI compute time.

Reply:
😂 😭